tests/fixtures/test_dashboard.py

```diff
@@ -13,3 +13,46 @@ def test_seller_dashboard(client, brief_responses, supplier_user):
     item = json.loads(response.data)['items'][0]
     assert item['name'] == 'I need a Developer'
     assert not item['is_downloaded']
+
+
+def test_buyer_dashboard_response(client, buyer_dashboard_briefs, users):
```

Comment:
small nitpick - I would be a bit more descriptive in these test names eg. what about the reposes are you testing?